lms/djangoapps/instructor_task/models.py

```diff
@@ -335,23 +335,23 @@ def links_for(self, course_id):
         can be plugged straight into an href
         """
         course_dir = self.key_for(course_id, '')
-<<<<<<< HEAD
-        sorted_link_data = sorted(
-            [
-                # each element in the array is a list with the following format: ('date, 'filename', 'url')
-                (key.key.split('/')[-1].split('_')[-1].split('.csv')[0], key.key.split('/')[-1], key.generate_url(expires_in=300))
-                for key in self.bucket.list(prefix=course_dir.key)
-            ],
-            reverse=True
-        )
-        return [
-            (link_data[1], link_data[2])
-            for link_data in sorted_link_data
-=======
+# TODO:FUNK <<<<<<< HEAD
+#         sorted_link_data = sorted(
+#             [
+#                 # each element in the array is a list with the following format: ('date, 'filename', 'url')
+#                 (key.key.split('/')[-1].split('_')[-1].split('.csv')[0], key.key.split('/')[-1], key.generate_url(expires_in=300))
+#                 for key in self.bucket.list(prefix=course_dir.key)
+#             ],
+#             reverse=True
+#         )
+#         return [
+#             (link_data[1], link_data[2])
+#             for link_data in sorted_link_data
+# TODO:FUNK =======
         return [
             (key.key.split("/")[-1], key.generate_url(expires_in=300))
             for key in sorted(self.bucket.list(prefix=course_dir.key), reverse=True, key=lambda k: k.last_modified)
->>>>>>> 00b75f0119b981641833240be214ef2076329747
+# TODO:FUNK >>>>>>> 00b75f0119b981641833240be214ef2076329747
```

Comment:
yes, @stvstnfrd we can use their logic. I just tested it locally for the LocalFSReportStore and it works as expected. We should accept upstream's changes above.